src/lib/zexe_backend/zexe_backend_common/scale_pack_round_5_wires.ml

```diff
@@ -3,38 +3,18 @@ open Core_kernel
 [%%versioned
 module Stable = struct
   module V1 = struct
-    type 'a t =
-      { xt: 'a
-      ; b: 'a
-      ; yt: 'a
-      ; xp: 'a
-      ; l1: 'a
-      ; yp: 'a
-      ; xs: 'a
-      ; ys: 'a
-      ; n1: 'a
-      ; n2: 'a }
+    type 'a t = {mutable xt: 'a; mutable b: 'a; mutable yt: 'a; mutable xp: 'a; l1: 'a; mutable yp: 'a; xs: 'a; ys: 'a; mutable n1: 'a; n2: 'a}
```

Comment:
CI will tell you to do this anyway, but: can you run `make reformat`?

Also, same comment as above about mutability.